exercises/queen-attack/description.md

```diff
@@ -1,27 +1,24 @@
 # Description
 
-Given the position of two queens on a chess board, indicate whether or not they
-are positioned so that they can attack each other.
+Given the position of two queens on a chess board, indicate whether or not they are positioned so that they can attack each other.
 
-In the game of chess, a queen can attack pieces which are on the same
-row, column, or diagonal.
+In the game of chess, a queen can attack pieces which are on the same row, column, or diagonal.
 
 A chessboard can be represented by an 8 by 8 array.
 
-So if you're told the white queen is at (2, 3) and the black queen at
-(5, 6), then you'd know you've got a set-up like so:
+So if you're told the white queen is at `c5` (zero-indexed at column 2, row 3) and the black queen at `f2` (zero-indexed at column 5, row 6), then you'd know you've got a set-up like so:
 
 ```text
-_ _ _ _ _ _ _ _
-_ _ _ _ _ _ _ _
-_ _ _ W _ _ _ _
-_ _ _ _ _ _ _ _
-_ _ _ _ _ _ _ _
-_ _ _ _ _ _ B _
-_ _ _ _ _ _ _ _
-_ _ _ _ _ _ _ _
+  a b c d e f g h
+8 _ _ _ _ _ _ _ _ 8
+7 _ _ _ _ _ _ _ _ 7
+6 _ _ _ _ _ _ _ _ 6
+5 _ _ W _ _ _ _ _ 5
+4 _ _ _ _ _ _ _ _ 4
+3 _ _ _ _ _ _ _ _ 3
+2 _ _ _ _ _ B _ _ 2
+1 _ _ _ _ _ _ _ _ 1
+  a b c d e f g h
 ```
 
-You'd also be able to answer whether the queens can attack each other.
-In this case, that answer would be yes, they can, because both pieces
-share a diagonal.
+You'd also be able to answer whether the queens can attack each other. In this case, that answer would be yes, they can, because both pieces share a diagonal.
```

Comment:
```suggestion
You'd also be able to answer whether the queens can attack each other. 
In this case, that answer would be yes, they can, because both pieces share a diagonal.
```

Reply:
You would or you could... no contraction here, as it becomes unclear, potentially.

Reply:
Actually think this makes more sense.